lib/turbolinks.rb

```diff
@@ -7,19 +7,27 @@
 require 'turbolinks/redirection'
 
 module Turbolinks
+  module ControllerConcern
```

Comment:
This is public now. Should we document what behavior will change in people's controllers or link them to documentation where they can see that?

I also don't like the Concern suffix here. Would `Turbolinks::Controller` be too broad?